### PR TITLE
Navbar fixes

### PR DESCRIPTION
**Pull request by johnsington**

### CODE REVIEW

File: templates/layout.jade [hunk 1]
```diff
@@ -38,35 +38,36 @@ html(class="no-js", lang="en", ng-app="monstercat")
...
+            label.monstercat Monstercat
 
           button.fa.fa-bars
 
           ul
             li.link
-              p
-                a(href="/") Home
+              a(href="/")
+                p Home
```
Comment (tmathews) [inline templates/layout.jade]: `p` tags are used for paragraphs, is there a reason we didn't wrap it in a span or a different tag and do some css targeting on the `ul > li > ...` to change the line-height or whatever was addressed here?

**Comment by tmathews** [comment]
Looks good other than that one issue.

File: templates/layout.jade (hunk 1 above)
Comment (johnsington) [inline templates/layout.jade]: It was what was in place before, but you're right since it's not semantically correct to use p tags. I'll change this.

Comment (tmathews) [inline templates/layout.jade]: Awesome, thanks!

**Pull request merged**